app/controllers/clearance/passwords_controller.rb

```diff
@@ -37,8 +40,14 @@ def update
 
   private
 
-  def deliver_email(user)
-    mail = ::ClearanceMailer.change_password(user)
+  def create_password_reset_for(user)
+    PasswordReset.create! do |password_reset|
```

Comment:
Hm, I don't think we want to use strong params here. IIRC, this was done so that we could associate the password reset with a user. I don't recall why I went for a block here instead of just `PasswordReset.create!(user: user)` though...so I'll just go ahead and simplify it to that.